frontend/public/components/_monitoring.scss

```diff
@@ -40,10 +40,6 @@ $monitoring-line-height: 18px;
   color: $color-pf-blue-400;
 }
 
-.form-control--silence-text {
```

Comment:
I think we want to keep this rule, just renamed to `pf-c-form-control--silence-text`.

Reply:
Good catch, Andy.  I removed the rule, but forgot to remove the class from the jsx.  

We don't actually want the class as it causes a layout bug with the new PF4 input class.
![Screen Shot 2019-07-12 at 8 52 44 AM](https://user-images.githubusercontent.com/895728/61129617-c5a2df00-a482-11e9-8973-55bb79628631.png)
Dropping the class gives us a consistent layout with the text area below..
![Screen Shot 2019-07-12 at 8 52 34 AM](https://user-images.githubusercontent.com/895728/61129644-cfc4dd80-a482-11e9-8a99-6a015f371163.png)

Reply:
👍 Thanks for the explanation @rhamilto

This does now differ from the mocks though. WDYT @cshinn? Are you happy with the 100% Creator input width, or should we bring back the max width like below?

![screenshot](https://user-images.githubusercontent.com/460802/61165978-77fda380-a562-11e9-9d12-0ab37daa8d95.png)